wordle-web/src/App.vue

```diff
@@ -1,28 +1,45 @@
-<template>
-  <header>
-    <div class="wrapper">
-      <nav>
-        <RouterLink to="/">Home</RouterLink> | <RouterLink to="/wordle">Wordle</RouterLink> |
-        <RouterLink to="/about">About</RouterLink>
-      </nav>
-      <v-btn @click="setInverseTheme"> Inverse Theme </v-btn>
-      <v-btn @click="setDarkTheme"> Dark Theme </v-btn>
-    </div>
-  </header>
-
-  <RouterView />
-</template>
-
 <script setup lang="ts">
-import { useTheme } from 'vuetify/lib/framework.mjs'
+import { RouterView } from 'vue-router'
+import ThemeSelector from './components/ThemeSelector.vue'
+</script>
 
-const theme = useTheme()
+<template>
+  <v-container fluid class="pa-0">
+    <v-row class="ma-0">
+      <v-col class="pa-0">
```

Comment:
I'd suggest not using v-row and v-col unless you are going to have multiple v-col elements for a grid system.